backend/src/app.js

```diff
@@ -0,0 +1,11 @@
+import express from 'express';
+import ProfileRoutes from './routes/profileRoutes.js';
+
+const app = express();
+
+app.use(express.json());
+
+app.use('/api/Profiles', ProfileRoutes);
+
+const PORT = process.env.PORT || 3000;
```

Comment:
Small one, but it breaks name consistency. This name doesn't match into any existing convention, therefore breaks all of them

Reply:
do you mean I should use camelCase instead of PascalCase here?

Reply:
Usually, it's camel case. `PORT` here is not PascalCase, cause in that case it would be `Port` (which is completely useless given that it has 1 word only). So keep it simple, just port from the application side, in .env it has to be PORT

Reply:
understood!